docs/pages/bare/installing-updates.mdx

```diff
@@ -65,37 +65,218 @@ Add the **"Supporting"** directory containing **Expo.plist** to your project in
+
+```
+
+```diff apps/bare-update/ios/bareupdate/AppDelegate.mm
+-  RCTBridge *bridge = [[RCTBridge alloc] initWithDelegate:self launchOptions:launchOptions];
++  self.launchOptions = launchOptions;
++  self.window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
++  #ifdef DEBUG
++    [self initializeReactNativeApp];
++  #else
++    EXUpdatesAppController *controller = [EXUpdatesAppController sharedInstance];
++    controller.delegate = self;
++    [controller startAndShowLaunchScreen:self.window];
++  #endif
++
++  [super application:application didFinishLaunchingWithOptions:launchOptions];
++
++  return YES;
++ }
++
++ - (RCTBridge *)initializeReactNativeApp
++ {
++  RCTBridge *bridge = [[RCTBridge alloc] initWithDelegate:self launchOptions:self.launchOptions];
+   RCTRootView *rootView = [[RCTRootView alloc] initWithBridge:bridge moduleName:@"main" initialProperties:nil];
+   id rootViewBackgroundColor = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"RCTRootViewBackgroundColor"];
+   if (rootViewBackgroundColor != nil) {
+```
+
+```diff apps/bare-update/ios/bareupdate/AppDelegate.mm
+     rootView.backgroundColor = [UIColor whiteColor];
+   }
+
+-  self.window = [[UIWindow alloc] initWithFrame:[UIScreen mainScreen].bounds];
+   UIViewController *rootViewController = [UIViewController new];
+   rootViewController.view = rootView;
+   self.window.rootViewController = rootViewController;
+   [self.window makeKeyAndVisible];
+
+-  [super application:application didFinishLaunchingWithOptions:launchOptions];
+-
+-  return YES;
++  return bridge;
+  }
+
+ - (NSArray<id<RCTBridgeModule>> *)extraModulesForBridge:(RCTBridge *)bridge
+```
+
+```diff apps/bare-update/ios/bareupdate/AppDelegate.mm
+  #ifdef DEBUG
+   return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index" fallbackResource:nil];
+  #else
+-  return [[NSBundle mainBundle] URLForResource:@"main" withExtension:@"jsbundle"];
++  return [[EXUpdatesAppController sharedInstance] launchAssetUrl];
+  #endif
+ }
+
++ - (void)appController:(EXUpdatesAppController *)appController didStartWithSuccess:(BOOL)success {
++  appController.bridge = [self initializeReactNativeApp];
++ }
+
+ // Linking API
+ - (BOOL)application:(UIApplication *)application openURL:(NSURL *)url options:(NSDictionary<UIApplicationOpenURLOptionsKey,id> *)options {
+   return [RCTLinkingManager application:application openURL:url options:options];
+```
+
+### Expo.plist
+
+```diff apps/bare-update/ios/bareupdate/Supporting/Expo.plist
+     <key>EXUpdatesURL</key>
+     <string>https://exp.host/@my-expo-username/my-app</string>
++    <key>EXUpdatesAutoSetup</key>
++    <false/>
+   </dict>
+ </plist>
+```
 
 ## Customizing Automatic Setup for Android
 
 By default, `expo-updates` requires no additional setup. If you want to customize the installation, e.g. to enable updates only in some build variants, you can instead follow these manual setup steps and then apply any customizations.
```

Comment:
```suggestion
By default, `expo-updates` requires no additional setup. If you want to customize the installation, for example, to enable updates only in some build variants, you can instead follow these manual setup steps and then apply any customizations.
```

Unrelated to the changes in this PR but I think we can replace the latin abbreviation here as per our [styleguide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#abbreviations).